Firestore/Source/Auth/FSTCredentialsProvider.m

```diff
@@ -108,24 +104,24 @@ - (void)getTokenForcingRefresh:(BOOL)forceRefresh
   // FIRFirestoreErrorCodeAborted error) if there is a user change while the request is outstanding.
   int initialUserCounter = self.userCounter;
 
-  void (^getTokenCallback)(NSString *, NSError *) = ^(NSString *_Nullable token,
-                                                      NSError *_Nullable error) {
-    @synchronized(self) {
-      if (initialUserCounter != self.userCounter) {
-        // Cancel the request since the user changed while the request was outstanding so the
-        // response is likely for a previous user (which user, we can't be sure).
-        NSDictionary *errorInfo = @{ @"details" : @"getToken aborted due to user change." };
-        NSError *cancelError = [NSError errorWithDomain:FIRFirestoreErrorDomain
-                                                   code:FIRFirestoreErrorCodeAborted
-                                               userInfo:errorInfo];
-        completion(nil, cancelError);
-      } else {
-        FSTGetTokenResult *result =
-            [[FSTGetTokenResult alloc] initWithUser:self.currentUser token:token];
-        completion(result, error);
-      }
-    };
-  };
+  void (^getTokenCallback)(NSString *, NSError *) =
```

Comment:
$ clang-format --version
clang-format version 5.0.0 (tags/google/stable/2017-06-22)

I ran ./scripts/style.sh and then discarded changes made to files that weren't touched in this patch. (However, I kept all changes within files that I did touch, even when those changes were in otherwise untouched blocks.)

Reply:
I have the same version as @rsgowman:

```
$ brew upgrade clang-format
Error: clang-format 2017-06-22 already installed
$ clang-format --version
clang-format version 5.0.0 (tags/google/stable/2017-06-22)
```

When I run against a clean checkout of master `scripts/style.sh` changes no files.

Reply:
Oh, I see the problem: 'cat style.sh | grep Auth' includes '-name 'Auth' -prune -o \'. So we're not linting anything in Auth directories. I suspect that was meant to target 'Firebase/Auth', but it's also targeting 'Firestore/Source/Auth'. (Or is this WAI?)

In my case, I ran style.sh not from the root, but from within the Auth directory, therefore bypassing the filter.

Reply:
Good find! The script is only intended to bypass `Firebase/Auth`.

Reply:
In that case, I'll leave the style fixes in (and will look into un-filtering Firestore/Source/Auth as well as looking for other unintended duplicates.)

Reply:
b/69924409

Reply:
Fix in https://github.com/firebase/firebase-ios-sdk/pull/508